ord_schema/units.py

```diff
@@ -79,11 +79,11 @@
         reaction_pb2.Wavelength.WAVENUMBER: ['cm-1', 'wavenumber', '1/cm'],
```

Comment:
Please ignore unless you enjoy this: "cm⁻¹".

Reply:
Done!